src/plugins/sync-plugin.js

```diff
@@ -424,10 +467,13 @@ const createNodeFromYElement = (el, schema, mapping, snapshot, prevSnapshot, com
       }
     }
   }
-  if (snapshot === undefined || prevSnapshot === undefined) {
-    el.toArray().forEach(createChildren)
-  } else {
-    Y.typeListToArraySnapshot(el, new Y.Snapshot(prevSnapshot.ds, snapshot.sv)).forEach(createChildren)
+  let isKnownNode = !!schema.nodes[el.nodeName]
+  if (isKnownNode) {
```

Comment:
You could simply do `if (schema.nodes[el.nodeName])`.

But you could simplify this even more by doing the following right at the beginning of `createNodeFromYElement`.

```js
if (!schema.nodes[el.nodeName]) {
  let fallbackNode = schema.node(el.nodeName, attrs, children);
  mapping.set(el, fallbackNode);
  return fallbackNode
}
```